pkg/crdconversion/crdconversion.go

```diff
@@ -26,16 +26,27 @@ const (
 
 	// healthPort is the port on which the '/healthz` requests are served
 	healthPort = 9095
+
+	// paths to convert CRD's
+	trafficAccessConverterPath         = "/convert/trafficaccess"
+	httpRouteGroupConverterPath        = "/convert/httproutegroup"
+	meshConfigConverterPath            = "/convert/meshconfig"
+	multiclusterServiceConverterPath   = "/convert/multiclusterservice"
+	egressPolicyConverterPath          = "/convert/egresspolicy"
+	trafficSplitConverterPath          = "/convert/trafficsplit"
+	tcpRoutesConverterPath             = "/convert/tcproutes"
+	ingressBackendsPolicyConverterPath = "/convert/ingressbackendspolicy"
```

Comment:
Thanks. In the future we could expose other endpoints such as `/health` , `/debug` if necessary, so this looks clean.